curator/run.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/bash
 
+TIMES=60
+
+function waitForES() {
+  for ((i=1; i<=$TIMES; i++ )); do
+    # test for ES to be up first
+    result=$(curl --cacert $ES_CA --key $ES_CLIENT_KEY --cert $ES_CLIENT_CERT -s -w "%{http_code}" -XGET "https://$ES_HOST:$ES_PORT/" -o /dev/null)
```

Comment:
I think we should address that in a different commit/PR - The change to allow `http` instead of only `https` will hit a lot of code.